shell/lisa_shell

```diff
@@ -30,8 +30,13 @@ LISA_PYTHON=${LISA_PYTHON:-python3}
 # By default use internal libraries
 export LISA_DEVMODE=${LISA_DEVMODE:-1}
 
+# By default use Lisa's PS1 and colorscheme for the shell
+export LISA_SHELL_QUIET=${LISA_SHELL_QUIET:-0}
```

Comment:
Being pedantic here, but the name irks me somewhat. Yes, you don't get the banner dumped, but you still get some other information dumped. It's about impacting the current shell.

Naming is hard though. `LISA_NO_COLORS`, `LISA_SHELL_PRISTINE`...?

Reply:
Yeah I suck at naming things, so whatever you think is best will work for me ;)

Reply:
The latest word shuffle that came out of my head is `LISA_PRESERVE_SHELL`, shall we go for that?

Reply:
Yep, sounds good